packages/host/app/components/operator-mode/code-submode.gts

```diff
@@ -668,12 +686,35 @@ export default class CodeSubmode extends Component<Signature> {
                       data-test-card-resource-loaded
                     />
                   {{else if this.selectedCardOrField}}
-                    <SchemaEditorColumn
-                      @file={{this.readyFile}}
-                      @card={{this.selectedCardOrField.cardOrField}}
-                      @cardTypeResource={{this.selectedCardOrField.cardType}}
-                      @openDefinition={{this.openDefinition}}
-                    />
+                    <Accordion class='accordion' as |A|>
```

Comment:
```suggestion
                    <Accordion as |A|>
```

Also an existing thing but this class isn’t doing anything and is already present on the component anyway:

<img width="543" alt="Boxel 2023-11-30 17-30-06" src="https://github.com/cardstack/boxel/assets/43280/26750eb2-2a53-4154-8b43-002e2acfac38">